addons/l10n_it_edi/models/account_edi_format.py

```diff
@@ -371,6 +371,11 @@ def _update_invoice_from_binary(self, filename, content, extension, invoice):
                 return self._import_fattura_pa(decoded_content, invoice)
         return super()._update_invoice_from_binary(filename, content, extension, invoice)
 
+    def _convert_date_from_xml(self, xsdate_str):
+        """ Dates in FatturaPA are ISO 8601 date format, pattern '[-]CCYY-MM-DD[Z|(+|-)hh:mm]' """
+        xsdate_pattern = r"-?(?P<date>-?\d{4}-\d{2}-\d{2})(?P<tz>[zZ]|[+-]\d{2}:\d{2})?"
+        return datetime.strptime(re.match(xsdate_pattern, xsdate_str).group("date"), DEFAULT_FACTUR_ITALIAN_DATE_FORMAT)
```

Comment:
@lordkrandel Is there an easy way to make sure it won't trigger errors (e.g. maybe it already falls back to False?)

Reply:
Done.  👍🏻